doc/indices.sgm

```diff
@@ -65,6 +83,22 @@
 <![CDATA[CREATE INDEX test_pos_idx ON test USING GIST (pos);]]>
 <![CDATA[VACUUM ANALYZE test;]]>
           </programlisting>
+          <para>
+            BRINs can be created through the following syntax: 
+          </para>
+          <programlisting>
+<![CDATA[CREATE INDEX test_pos_idx USING BRIN ON test (pos);]]>
+          </programlisting>
+          <para>
+            By default, BRINs summarize block of 128 pages. The lower numbers
+            of pages are specified, the higher granularity is reached during
+            the searches, and performance's gap between GiST indexes and BRINs
+            is lower (consider that BRINs size increases as well). Different
+            summarizations can be used with the following command:
```

Comment:
This whole paragraph (lines 93-97) could be better phrased. Here is what I recommend:

By default, BRINs summarize blocks of 128 pages. The smaller the number of pages specified, the higher the granularity in searches, and the gap in performance between GiST indexes and BRINs will be decreased. (Note that the size of the BRINs increases as well.) Different summarizations can be used with the following command:

Reply:
Yes, it does make sense. I was thinking if it makes sense as well to add a reference to the fact that data in the pages should have some kind of "natural clustering" in the pages themselves: since the Block Range INdex keep the summarised information about the MBR that includes all the geometries in single page, then in case the geometries are widely distributed in the pages it will result for most of them an MBR which is pretty identical to the MBR of the whole dataset, making the index useless. This might happen for example when the geometries are imported in bulk in the DB through parallel importer.

In this case, the solution is to use a GiST index to CLUSTER the table at the end of the bulk import, operation that can take some time for very large DBs but it needs to be done only once. 

For future inserts, autovacuum takes care for regular maintenance of the index with unsummarized pages. Further details here: https://www.postgresql.org/docs/current/brin-intro.html

Reply:
@esabol Not sure, the phrase "(Note that the size of the BRINs increases as well.) " with braces. May be, just remote braces? It will be a regular sentense.